### PR TITLE
Added new dialog to edit cell contents in Data View

### PR DESCRIPTION
**Pull request by derekagorhom**
Fixes partly #8239 
Replaces #8264 

This PR is ready for Review

@rdstern , @N-thony can you review/Test this PR 

Thanks

### CODE REVIEW

**Review by rdstern — DISMISSED** [review]
@derekagorhom great.  @lloyddewit once you are ok I would be happy to see this merged.

File: instat/dlgEdit.vb [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+
+    Private Sub ucrBase_ClickReset(sender As Object, e As EventArgs) Handles ucrBase.ClickReset
+        SetDefaults()
+        SetRCodeForControls(True)
+        TestOKEnabled()
+    End Sub
+
+    Private Sub SetSelectedColumn()
+        ucrReceiverName.Add(strSelectedColumn)
+        ucrRowNumber.Add(strRowText)
```
Comment (lloyddewit) [inline instat/dlgEdit.vb:107]: Does this control show the cell's row number or the cell's contents?
If the 2nd, then should this be renamed?

File: instat/dlgEdit.vb [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+        ucrReceiverRow.SetName(StrRowIndex)
+        ucrNewName.SetName(strRowText)
+        bUseSelectedColumn = False
+    End Sub
+
+    Private Sub ucrCoreControls_ControlContentsChanged() Handles ucrNewName.ControlContentsChanged, ucrReceiverName.ControlContentsChanged, ucrReceiverRow.ControlContentsChanged, ucrInputRows.ControlContentsChanged, ucrRowNumber.ControlContentsChanged
+        TestOKEnabled()
+    End Sub
+
+    Public Sub SetCurrentColumn(strColumn As String, strRowNumber As String, strIRow As String)
```
Comment (lloyddewit) [inline instat/dlgEdit.vb:117]: Is `strRowNumber` the row number or row contents?

File: instat/dlgEdit.vb [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+        bUseSelectedColumn = True
+    End Sub
+
+    Private Sub ucrReceiverName_ControlValueChanged(ucrChangedControl As ucrCore) Handles ucrReceiverName.ControlValueChanged, ucrInputRows.ControlValueChanged
+        If Not ucrReceiverName.IsEmpty Then
+            If Not ucrInputRows.IsEmpty Then
+                ucrNewName.Visible = False
+            Else
+                ucrNewName.Visible = True
+            End If
```
Comment (lloyddewit) [inline instat/dlgEdit.vb:130]: simplify?
```suggestion
            ucrNewName.Visible = ucrInputRows.IsEmpty
```

File: instat/dlgEdit.vb (hunk 2 above)
Comment (derekagorhom) [reply instat/dlgEdit.vb]: @lloyddewit I changed it to `strRowContents` is that ok for you?

**Review by rdstern — APPROVED** [review]
@lloyddewit there are no visible changes and it still all looks fine to me.

**Comment by lloyddewit** [comment]
@derekagorhom Thanks for the changes, most of my comments are noe resolved. There is still one open comment above, please could you take a look? thanks

File: instat/dlgEdit.vb (hunk 1 above)
Comment (derekagorhom) [reply instat/dlgEdit.vb]: @lloyddewit I renamed these controls

Comment (derekagorhom) [reply instat/dlgEdit.vb]: image
`ucrReceiverRow` now shows cell content while `ucrRowNumber` shows the row number

Comment (lloyddewit) [reply instat/dlgEdit.vb]: Understood, thanks

**Review by lloyddewit — APPROVED** [review]

**Comment by lloyddewit** [comment]
@derekagorhom Thanks, I will merge (there's currently a release being built so I'll merge when the release build is done).

**Pull request merged**